pkg/obfuscator/ip_test.go

```diff
@@ -32,7 +32,7 @@ func TestIPObfuscatorStatic(t *testing.T) {
 			name:   "ipv4 in aws etcd pathing",
 			input:  "must-gather/etcd-ip-10-0-187-218.ec2.internal/some.yaml",
 			output: "must-gather/etcd-ip-xxx.xxx.xxx.xxx.ec2.internal/some.yaml",
-			report: map[string]string{"10-0-187-218": obfuscatedStaticIPv4, "10.0.187.218": obfuscatedStaticIPv4},
+			report: map[string]string{"10-0-187-218": obfuscatedStaticIPv4},
```

Comment:
okay so asMap takes the occurrence, right? +1

Reply:
The opposite. AsMap is just the key and replacedWith. it is generated from a structure that for this example is:
```
replacements:
  - canonical: 10.0.187.218
    replacedWith: x-ipv4-0000000001-x
    occurrences:
      - original: 10-0-187-218
        count: 1
```
AsMap returns `.occurences[].original: .replacedWith`